src/pages/Bridge/RedeemForm/index.tsx

```diff
@@ -364,22 +417,52 @@ const RedeemForm = (): JSX.Element | null => {
               wrappedTokenSymbol: WRAPPED_TOKEN_SYMBOL
             })}
           </FormTitle>
-          <TokenField
-            id={WRAPPED_TOKEN_AMOUNT}
-            label={WRAPPED_TOKEN_SYMBOL}
-            min={0}
-            {...register(WRAPPED_TOKEN_AMOUNT, {
-              required: {
-                value: true,
-                message: t('redeem_page.please_enter_amount')
-              },
-              validate: (value) => validateForm(value)
-            })}
-            approxUSD={`≈ ${displayMonetaryAmountInUSDFormat(parsedInterBTCAmount || BitcoinAmount.zero(), usdPrice)}`}
-            error={!!errors[WRAPPED_TOKEN_AMOUNT]}
-            helperText={errors[WRAPPED_TOKEN_AMOUNT]?.message}
-          />
+          <div>
+            <AvailableBalanceUI
+              label={t('redeem_page.maximum_in_single_request')}
+              balance={displayMonetaryAmount(maxRedeemableCapacity)}
+              tokenSymbol={WRAPPED_TOKEN_SYMBOL}
+            />
+            <TokenField
+              id={WRAPPED_TOKEN_AMOUNT}
+              label={WRAPPED_TOKEN_SYMBOL}
+              min={0}
+              {...register(WRAPPED_TOKEN_AMOUNT, {
+                required: {
+                  value: true,
+                  message: t('redeem_page.please_enter_amount')
+                },
+                validate: (value) => validateForm(value)
+              })}
+              approxUSD={`≈ ${displayMonetaryAmountInUSDFormat(
+                monetaryWrappedTokenAmount || BitcoinAmount.zero(),
+                usdPrice
+              )}`}
+              error={!!errors[WRAPPED_TOKEN_AMOUNT]}
+              helperText={errors[WRAPPED_TOKEN_AMOUNT]?.message}
+            />
+          </div>
           <ParachainStatusInfo status={parachainStatus} />
+          {!premiumRedeemSelected && (
```

Comment:
With the premium redeem selected, I disabled the vault selection logic in order to make things simple and the feature scoped.
![demo](https://user-images.githubusercontent.com/84005068/206886492-43bcf929-667b-4a92-9497-e8afb1849f49.gif)

Reply:
Can you confirm there is no use case for manually selecting a premium redeem vault?

Reply:
As far as I remember, there might be but in that case, it needs to be newly implemented on top of the current one.
One of the main reasons why I have not implemented for the premium redeem is that it's tricky to test it out since premium redeem case does not always happen.
I'd suggest implementing it via another dedicated ticket.